### PR TITLE
Update deck.gl to 8.5.0 + small material-ui changes

### PR DESCRIPTION
**Pull request by Clebal**

### CODE REVIEW

**Review by VictorVelarde — CHANGES_REQUESTED** [review]
Same, looks good, but you must also update the "@deck.gl/carto" references in **template.json**, not just in the .dev.json (just for developing). Please also add a note to changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Not released
 
+- Update deck.gl version to 8.5.0 [#264](https://github.com/CartoDB/carto-react-template/pull/264)
```
Comment (VictorVelarde) [inline CHANGELOG.md:5]: 👌🏻

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
...
+- Fix external libraries warnings [#264](https://github.com/CartoDB/carto-react-template/pull/264)
```
Comment (VictorVelarde) [inline CHANGELOG.md:6]: If you're gonna leave an independent line here, then try to be more explicit. Eg: "Fix material-ui warnings on justify property"

Comment (Clebal) [reply CHANGELOG.md:6]: Okey

**Review by VictorVelarde — APPROVED** [review]
🚀  Nice, thx!

**Pull request merged**